docs/read_the_docs/source/workflow_inputs/characteristics.rst

```diff
@@ -3,6 +3,27 @@
 Housing Characteristics
 =======================
 
+Each parameter sampled by the national project is listed alphabetically below.
+For each, the following (if applicable) are reported based on the contents of `source_report.csv <https://github.com/NREL/resstock/blob/develop/project_national/resources/source_report.csv>`_:
+
+- **Description**
+- **Created by**
+- **Source**
+- **Assumption**
```

Comment:
These should be semi-colon delimited. So we should separate these out into bullets.